### PR TITLE
feat: created isPalindrome function, minNum, maxNum functions, and re…

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,49 +1,94 @@
...
+//   if (arr3[i] % 2 == 0) {
+//     console.log(arr3[i]);
+//   }
+// }
+// //alternative
+// for (let i = 0; i < arr3.length; i++) {
+//   if ((arr3[i] & 1) == 0) {
+//     console.log(arr3[i]);
+//   }
+// }
```
Comment (Marinarium) [inline src/main.js]: Please, put it in another file or delete(your work will save on lesson-2 branch), don't leave an unused code=)

File: src/main.js [hunk 2]
```diff
@@ -1,49 +1,94 @@
...
+  restNumbers[0] = Math.floor(Math.random() * 100);
+  restNumbers[1] = Math.floor(Math.random() * 100);
+  restNumbers[2] = Math.floor(Math.random() * 100);
+  restNumbers[3] = Math.floor(Math.random() * 100);
+  restNumbers[4] = Math.floor(Math.random() * 100);
+  restNumbers[5] = Math.floor(Math.random() * 100);
+  restNumbers[6] = Math.floor(Math.random() * 100);
+  restNumbers[7] = Math.floor(Math.random() * 100);
+  restNumbers[8] = Math.floor(Math.random() * 100);
+  restNumbers[9] = Math.floor(Math.random() * 100);
```
Comment (Marinarium) [inline src/main.js]: try to do your code shorter
[DRY](https://www.youtube.com/watch?v=NWemqNMCesQ&ab_channel=SergeyNemchinskiy) 
image
[and more](https://www.fullstackguy.ru/blog/2022/04/dry-principle/)

File: src/main.js [hunk 3]
```diff
@@ -1,49 +1,94 @@
...
-    console.log(arr3[i]);
+function replaceZero(arr, find, replace) {
+  let arrayStr = arr.map((item) => item.toString());
+  let arrayResult = [];
+  for (let i = 0; i < arrayStr.length; i++) {
+    if (arrayStr[i] % 10 === 0) {
+      arrayResult.push(arrayStr[i].replace(find, replace));
+    } else arrayResult.push(+arrayStr[i]);
   }
+  return arrayResult;
```
Comment (Marinarium) [inline src/main.js:31]: your code doesn;t work with 100.  it should be 1zerozero

напишите функцию, которая будет заменять все 0 на строку 'zero';

how about a challange?) try to create such a function, that will be work even with 207 => '2zero7' =)

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (LudmilaZhuleha) [reply src/main.js]: Done.

File: src/main.js (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/main.js]: Done using ...Array().map()

File: src/main.js (hunk 3 above)
Comment (LudmilaZhuleha) [reply src/main.js:31]: Done.
Initial solution was with replace() method, with replace only the first entry. 
Final solution uses check for entries (includes()) and replaceAll() method replacing all the entries being found.

**Pull request merged**